src/test/app/AMM_test.cpp

```diff
@@ -5088,30 +5140,30 @@ struct AMM_test : public jtx::AMMTest
     void
     testCore()
     {
-        testInvalidInstance();
-        testInstanceCreate();
-        testInvalidDeposit();
-        testDeposit();
-        testInvalidWithdraw();
-        testWithdraw();
-        testInvalidFeeVote();
-        testFeeVote();
+        //        testInvalidInstance();
```

Comment:
Should uncomment the other tests.

Reply:
Hello @thejohnfreeman , I don't have permissions to push into your branch. Can you please incorporate greg's last three suggestions?

Reply:
Of course I'll work on addressing these comments (I opened the PR after all), but remember that you (and everyone) have permission to open a PR in my fork, against the branch of this PR (`amm-tests`). In my opinion, that's the way non-authors _should_ contribute changes to a PR. Don't forget the "suggest changes" feature in these review comments (see step 6 in [this section](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request)).